src/visualizations/features/VisualizationPicture.tsx

```diff
@@ -0,0 +1,29 @@
+import { gql } from "@apollo/client";
+import { VisualizationPicture_VisualizationFragment } from "./VisualizationPicture.generated";
+
+type VisualizationPictureProps = {
+  visualization: VisualizationPicture_VisualizationFragment;
+};
+
+const VisualizationPicture = (props: VisualizationPictureProps) => {
+  const { visualization } = props;
+  return (
+    <div className="flex h-10 w-10 flex-shrink-0 items-center text-gray-400">
```

Comment:
Why do we need the `text-gray-400`?

Reply:
I forgot to remote it.

Reply:
It was for testing purpose, forgot to remove it.

Reply:
It was for testing purpose, forgot to remove it.